content/posts/weekly/June-2022.md

```diff
@@ -9,3 +9,12 @@ weight: 61
 - Packit shows basic information about allowlisting in the status description
   when your namespace is not allowed.
   ([packit-service#1533](https://github.com/packit/packit-service/pull/1533))
+
+## Week 23 (June 7th - June 13th)
+
+- The creation of Bodhi updates should not time out anymore, because we no longer get the latest build of a package via Bodhi.
```

Comment:
```suggestion
- The creation of Bodhi updates should not time out anymore, because we no longer get the latest build of a package via Bodhi, but use Koji directly.
```
the message felt incomplete to me otherwise

Reply:
okay, I can see that Mato had an opposite suggestion, let's not change again: if people are interested, they can always open the PR